blockchain/blockdao.go

```diff
@@ -150,8 +156,47 @@ func (dao *blockDAO) getBlockHeight(hash hash.Hash32B) (uint64, error) {
 	return enc.MachineEndian.Uint64(value), nil
 }
 
+func getBlockPath(h uint64) string {
+	bytes := make([]byte, 8)
+	binary.BigEndian.PutUint64(bytes, h)
+
+	return path.Join(
+		blockNS,
+		hex.EncodeToString(bytes[:2]),
```

Comment:
Can we use decimal instead? Easy to find the block file manually.

Reply:
And no need to break it into multi dir level. Ext4/NTFS seems to allow all files in one dir.

Reply:
2^32 files are not enough, we need to at least split it into 2 directories